From ec32d54f74506571df97ba4a732fa2877e5f4886 Mon Sep 17 00:00:00 2001
From: Judith Mendez <jm@ti.com>
Date: Thu, 22 Feb 2024 12:39:45 -0600
Subject: [PATCH 05/27] mmc: sdhci_am654: Add ret to sdhci_am654_get_otap_delay
 function

The below patch adds missing OTAP/ITAP enable in the sdhci_am654
driver, but a ret was missed in sdhci_am654_get_otap_delay(). Add
now to fix OTAP/ITAP enable.

Fixes: 14951df5ad99 ("mmc: sdhci_am654: Add missing OTAP/ITAP enable")
Signed-off-by: Judith Mendez <jm@ti.com>
---
 drivers/mmc/host/sdhci_am654.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/host/sdhci_am654.c b/drivers/mmc/host/sdhci_am654.c
index c48080eaa545..38392b6f4100 100644
--- a/drivers/mmc/host/sdhci_am654.c
+++ b/drivers/mmc/host/sdhci_am654.c
@@ -729,8 +729,8 @@ static int sdhci_am654_get_otap_delay(struct sdhci_host *host,
 		}
 
 		if (td[i].itap_binding) {
-			device_property_read_u32(dev, td[i].itap_binding,
-						 &sdhci_am654->itap_del_sel[i]);
+			ret = device_property_read_u32(dev, td[i].itap_binding,
+						       &sdhci_am654->itap_del_sel[i]);
 
 			if (!ret)
 				sdhci_am654->itap_del_ena[i] = 0x1;
-- 
2.39.2

